packages/mrm/package.json

```diff
@@ -19,21 +19,19 @@
     "src"
   ],
   "dependencies": {
-    "cross-spawn": "7.0.3",
     "git-username": "^1.0.0",
     "glob": "^7.1.6",
     "inquirer": "^7.0.4",
     "is-directory": "^0.3.1",
     "kleur": "^3.0.3",
+    "libnpx": "^10.2.4",
     "listify": "^1.0.0",
     "lodash": "^4.17.15",
     "longest": "^2.0.1",
     "middleearth-names": "^1.1.0",
     "minimist": "^1.2.0",
     "mrm-core": "^4.1.3",
     "mrm-preset-default": "^2.2.5",
-    "package-json": "6.5.0",
-    "requireg": "^0.2.2",
```

Comment:
If I'm not mistaken, if the task or preset is not found in the given locations, it'll try to install with `npx`. So I guess the answer is no, but it won't break.

Reply:
I think npx uses global or local packages if they are installed but it's worth checking, and likely documenting.

Reply:
`npx` does check local packages first (not sure about globals) but we're using the `_ensurePackages` function, which does not do that. I checked if that part could be simply added, but seems like there are hacks on top of hacks and too much code for little gain.

I can take another look but my impression is that people don't use globally installed packages anymore. For local packages we can first check `CWD/node_modules` first.

Reply:
Then it's going to be a breaking change and a major release, which isn't a big problem.

The main problem I see is that npx is very very slow even if the package is cached already, so running an alias with a dozen of tasks could take a lot of time. If we can solve this problem, then it should be fine. I wanted to remove the default preset from the mrm package in the next major release anyway.

Reply:
Glad to hear that.

I'm thinking that I should be able to not resort to `npx` at least for locally installed packages, and will look into it now.

Reply:
The last commit I made makes this possible.

Reply:
I just noticed that this makes the behavior very weird, since we first check `taskPackageName` locally and then on npm registry, then we switch to `taskName`. The order should be:
- taskPackageName locally
- taskName locally
- taskPackageName on npm
- taskName on npm

I'll revert the last change and fix this.

Reply:
Should be fixed now.